src/util/error.cpp

```diff
@@ -14,7 +14,7 @@ std::string TransactionErrorString(const TransactionError err)
         case TransactionError::OK:
             return "No error";
         case TransactionError::MISSING_INPUTS:
-            return "Missing inputs";
+            return "Inputs missing or spent";
```

Comment:
Cleaning these error messages could probably be its own commit.

Reply:
Yeah, agreed, will fix.